pymc/tests/test_distributions_moments.py

```diff
@@ -434,3 +440,83 @@ def test_logistic_moment(mu, s, size, expected):
     with Model() as model:
         Logistic("x", mu=mu, s=s, size=size)
     assert_moment_is_expected(model, expected)
+
+
+@pytest.mark.parametrize(
+    "get_moment, size, expected",
+    [
+        (None, None, 0.0),
+        (None, 5, np.zeros(5)),
+        ("custom_moment", None, 5),
+        ("custom_moment", (2, 5), np.full((2, 5), 5)),
+    ],
+)
+def test_density_dist_moment(get_moment, size, expected):
```

Comment:
```suggestion
def test_density_dist_default_moment_univariate(get_moment, size, expected):
```

And I think you can remove the "custom_moment" checks as they are already covered in the tests below